### PR TITLE
Electron-142 (Build Number)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
Included build version in the about app window.

screen shot 2017-10-30 at 2 28 49 pm

@VikasShashidhar , @VishwasShashidhar & @lneir Please review. Thanks

### CODE REVIEW

File: js/aboutApp/renderer.js [hunk 1]
```diff
@@ -9,13 +9,19 @@ renderDom();
...
         copyright.innerHTML = `Copyright &copy; ${new Date().getFullYear()} ${applicationName}`
     });
 }
+
+ipcRenderer.on('buildVersion', (event, buildVersion) => {
+    let versionText = document.getElementById('version');
+    const version = remote.app.getVersion();
+
+    if (versionText) {
+        versionText.innerHTML = version ? `Version ${version} (${version}.${buildVersion})` : null;
```
Comment (VishwasShashidhar) [inline js/aboutApp/renderer.js]: Instead of `null`, let's make this `N/A`

File: package.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "name": "Symphony",
   "productName": "Symphony",
   "version": "2.0.0",
+  "buildVersion": "",
```
Comment (VishwasShashidhar) [inline package.json]: Can we also call this "buildNumber" in place of "buildVersion".

**Comment by VishwasShashidhar** [comment]
lgtm

File: js/aboutApp/renderer.js [hunk 3]
```diff
@@ -9,13 +9,19 @@ renderDom();
...
         copyright.innerHTML = `Copyright &copy; ${new Date().getFullYear()} ${applicationName}`
     });
 }
+
+ipcRenderer.on('buildNumber', (event, buildNumber) => {
+    let versionText = document.getElementById('version');
+    const version = remote.app.getVersion();
+
+    if (versionText) {
+        versionText.innerHTML = version ? `Version ${version} (${version}.${buildNumber})` : 'N/A';
```
Comment (VikasShashidhar) [inline js/aboutApp/renderer.js]: Does this work for Mac and Windows?

Comment (KiranNiranjan) [reply js/aboutApp/renderer.js]: This feature is only for windows as we are restricting it in the menu template here: https://github.com/symphonyoss/SymphonyElectron/blob/8fb57d23759da6582c81ef0f3f362fdace185ec9/js/menus/menuTemplate.js#L243

**Pull request merged**